src/components/BuildStartButton/BuildStartButton.tsx

```diff
@@ -186,7 +186,7 @@ export const BuildStartButton = ({ buildConfig, groupConfig, size = 'md' }: IBui
   };
 
   return (
-    <>
+    <span>
```

Comment:
This is to remove the gap between the build button and the dropdown list button caused by the PageLayout actions, which will also benifit NCL-7205